### PR TITLE
.github/workflows: Enable standard CI build tests for fuzzing

### PR DESCRIPTION
**Pull request by benjamindoron**
Fuzzing targets are excluded from the workspace, so, run build tests on them explicitly.

Note that Clippy is expected to fail until #681 is resolved.

### CODE REVIEW

**Comment by benjamindoron** [comment]
DNM: Not all targets have been merged first.

**Comment by benjamindoron** [comment]
@korran @jhand2 See here for running tests on the targets.

**Comment by jhand2** [comment]
LGTM. I'll sign off once the non-CI fuzzing PRs are merged.

**Comment by benjamindoron** [comment]
This is blocked on https://github.com/chipsalliance/caliptra-sw/pull/811 and https://github.com/chipsalliance/caliptra-dpe/pull/185. Afterwards I think it can be merged

**Review by jhand2 — APPROVED** [review]

**Comment by jhand2** [comment]
LGTM, merging. Note that I removed "[DNM]" from the final commit message. Just for history, we may also want to edit this PR name to also remove that.

**Comment by jhand2** [comment]
> LGTM, merging. Note that I removed "[DNM]" from the final commit message. Just for history, we may also want to edit this PR name to also remove that.

It turns out I'm able to do that myself 😄 Done.

**Pull request merged**
